<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToMaintenance</class>
 <widget class="QDialog" name="AddToMaintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modelBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startDateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="deadlineDateLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Deadline Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deadlineDateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Assign</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
